challenge_storing_groceries/src/challenge_storing_groceries/open_door.py

```diff
@@ -0,0 +1,197 @@
+# System
+import os
+
+# ROS
+import math
+import rospy
+import smach
+import tf2_ros
+import tf2_geometry_msgs  # required for transforms
+from geometry_msgs.msg import Twist, Vector3, PoseStamped, Quaternion
+from robot_skills.amigo import Amigo
+# TU/e
+from robot_skills.util.kdl_conversions import VectorStamped
+
+# Challenge storing groceries
+from tf.transformations import euler_from_quaternion, quaternion_from_euler
+
+import config
+
+import robot_smach_states as states
+
+import robot_smach_states.util.designators as ds
+
+
+def _clamp(abs_value, value):
+    return max(-abs_value, min(abs_value, value))
+
+
+def _get_yaw_from_quaternion_msg(msg):
+    """
+    Returns the yaw angle from a rotation in quaternion representation (msg)
+    :param msg: The quaternion msg
+    :return: Yaw angle
+    """
+    orientation_list = [msg.x, msg.y, msg.z, msg.w]
+    _, _, yaw = euler_from_quaternion(orientation_list)
+    return yaw
+
+
+def _wrap_angle_pi(angle):
+    """
+    Wraps between -pi and +pi
```

Comment:
Preferably one of the limits is not included, so (-pi, pi]

Reply:
No time to check and test this, I'm not going to change it.